public/server/db/ux.json

```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "User Experience"
```

Comment:
Abbreve is an unofficial dictionary for slang. Unfortunately, abbreviations like these don't fit in. Hope you understand.